platform/platform-configuration-migration/src/test/java/org/codice/ddf/configuration/admin/ConfigurationAdminMigrationTest.java

```diff
@@ -622,13 +580,12 @@ void moveFile(Path source, Path destination) throws IOException {
     @Test
     public void testNotifyWithValidFile() throws Exception {
```

Comment:
I didn't change this function, so I stepped through the code, and it is tesing notify with a valid file, i don't think it gets reprocessed, im fine leaving the test name as it is